code/modules/mod/mod_theme.dm

```diff
@@ -628,20 +628,21 @@
 	charge_drain = DEFAULT_CHARGE_DRAIN * 1.5
 	slowdown_deployed = 0.5
 	allowed_suit_storage = list(
+		/obj/item/crowbar/power/paramedic,
+		/obj/item/defibrillator/compact,
```

Comment:
Fikou asked me to put this on the medical MOD
I thought it was redundant given the defibrillator module but I trust John Modsuit